### DISCUSSION TITLE
Authentication with Google Cloud Resource Service Account

### DISCUSSION THREAD

**Discussion by michaelaguiar** [spatie/laravel-google-cloud-storage#57] [Q&A]
I have an application on Google Cloud Run with an attached service account. I set the permissions for this service account to access cloud storage, and it works locally (using keys generated from same service account). However, when I deploy my code, I get a 401 Unauthenticated error when trying to store file in cloud storage FROM cloud run.

It seems that it's not using the service account attached to the Cloud Run instance. Is there anything specific I have to do to get this to work?

**Comment by azad677677** [comment]
The goal is not to exchange my counterfeit currency for dollars, but to give way to those counterfeits and release my locked money.
